.devcontainer/dotnet/devcontainer.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "Dotnet",
+    "image": "mcr.microsoft.com/devcontainers/dotnet:8.0",
+    "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}/server/dotnet",
+    "forwardPorts": [
+        5000,
+        5001
+    ],
+    "portsAttributes": {
+        "5000": {
+            "label": "Dotnet HTTP",
+            "onAutoForward": "openBrowserOnce"
+        },
+        "5001": {
+            "label": "Dotnet HTTPS",
+            "onAutoForward": "openBrowserOnce"
+        }
+    },
+    "secrets": {
+        "BRAINTREE_MERCHANT_ID": {
+            "description": "Sandbox merchant ID of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        },
+        "BRAINTREE_PUBLIC_KEY": {
+            "description": "Sandbox public key of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
+        },
+        "BRAINTREE_PRIVATE_KEY": {
+            "description": "Sandbox private key of the application.",
+            "documentationUrl": "https://www.braintreepayments.com/sandbox"
```

Comment:
Is URL should be https://sandbox.braintreegateway.com/ ? 
This will take to sandbox dashboard home page.